lib/app/service/nekoton_related/connection_service/connection_service.dart

```diff
@@ -66,23 +64,31 @@ class ConnectionService {
     );
 
     await _nekotonRepository.updateTransport(
-      _createStrategyByType(type, transport),
+      _createStrategyByType(transport, connection),
```

Comment:
Maybe rename method to `_createStrategyByConnection` ?

Reply:
done